vllm/lora/layers.py

```diff
@@ -938,7 +938,7 @@ def soft_cap(self):
 
     @property
     def use_gather(self):
-        return self.base_layer.use_gather
+        return not self.base_layer.use_all_gather
```

Comment:
The function name should be updated to `use_all_gather`? And why it's now `not` use all gather?

Reply:
fixed in [33cc388](https://github.com/vllm-project/vllm/pull/12071/commits/33cc388107e6eab262da894e7b2e2b8430f9c277)